addons/hr_payroll/tests/test_benefit.py

```diff
@@ -2,22 +2,21 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from datetime import datetime
-import pytz
 from dateutil.relativedelta import relativedelta
-from odoo import exceptions
+import pytz
+
 from odoo.fields import Datetime, Date
 from odoo.tests.common import tagged
 from odoo.addons.hr_payroll.tests.common import TestPayslipBase
 
 
 @tagged('benefit')
 class TestBenefit(TestPayslipBase):
-
     def setUp(self):
         super(TestBenefit, self).setUp()
         self.tz = pytz.timezone(self.richard_emp.tz)
-        self.start = self.to_datetime_tz('2015-11-01 01:00:00')
-        self.end = self.to_datetime_tz('2015-11-30 23:59:59')
+        self.start = datetime(2015, 11, 1, 1, 0, 0)
+        self.end = datetime(2015, 11, 30, 23, 59, 59)
         self.resource_calendar_id = self.env['resource.calendar'].create({'name': 'Zboub'})
         self.env['hr.contract'].create({
             'date_start': self.start - relativedelta(days=5),
```

Comment:
Beware: `date_start` is a date field, to one should localize the value in `self.tz`, then take the corresponding `date()`.